scripts/build-ui.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+git submodule update --remote
+BASEDIR=$(git rev-parse --show-toplevel)
+UI_DIR="$BASEDIR/pkg/ui/pd-web"
+TARGET_DIST="$BASEDIR/pkg/ui/dist"
+SCRIPTS_DIR="$BASEDIR/scripts"
+echo "##### Build web UI"
+cd $UI_DIR
+yarn
+PUBLIC_URL=/ui/ yarn build
```

Comment:
```suggestion
PUBLIC_URL=/ui yarn build
```

Not recommended to end with `/`